qbft/instance.go

```diff
@@ -47,6 +49,10 @@ func NewInstance(
 	}
 }
 
+func (i *Instance) ForceStop() {
```

Comment:
When I saw `ForceStop` I got the impression there's a regular `Stop` and force is kinda stronger. Why not call it just `Stop`?

Reply:
There is a regular stop if we hit round 15

Reply:
I see. it just that we not proceed after round 15, but there's not active "stop"

Reply:
not proceed means we stop I think 🤷